File: CS-Examples/11_Formatting/MakeCellActive.cs
using Spire.Xls;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace MakeCellActive
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            // Create a workbook
            Workbook workbook = new Workbook();

            // Load file froom disk
            workbook.LoadFromFile(@"..\..\..\..\..\..\Data\templateAz.xlsx");

            // Get the 2nd sheet
            Worksheet sheet =  workbook.Worksheets[1];

            // Set the 2nd sheet as an active sheet.
            sheet.Activate();

            // Set B2 cell as an active cell in the worksheet.
            sheet.SetActiveCell(sheet.Range["B2"]);

            // Set the B column as the first visible column in the worksheet.
            sheet.FirstVisibleColumn = 1;

            // Set the 2nd row as the first visible row in the worksheet.
            sheet.FirstVisibleRow = 1;

            // Specify the name for the resulting Excel file
            String result = "MakeCellActive_result.xlsx";

            // Save the workbook to the specified file in Excel 2010 format
            workbook.SaveToFile(result, ExcelVersion.Version2010);

            // Dispose of the workbook object to release resources
            workbook.Dispose();

            // Launch the document
            FileViewer(result);
        }

        private void FileViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }
    }
}

File: CS-Examples/11_Formatting/MakeCellActive.vb
Imports Spire.Xls
Imports System.ComponentModel
Imports System.Text

Namespace MakeCellActive
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
            ' Create a new instance of Workbook
            Dim workbook As New Workbook()

            ' Load the Excel file from the specified path
            workbook.LoadFromFile("..\..\..\..\..\..\Data\templateAz.xlsx")

            ' Get the first worksheet in the workbook
            Dim sheet As Worksheet = workbook.Worksheets(1)

            ' Activate the worksheet
            sheet.Activate()

            ' Set the active cell to cell B2
            sheet.SetActiveCell(sheet.Range("B2"))

            ' Set the first visible column to column 1
            sheet.FirstVisibleColumn = 1

            ' Set the first visible row to row 1
            sheet.FirstVisibleRow = 1

            ' Specify the filename for the resulting Excel file
            Dim result As String = "MakeCellActive_result.xlsx"

            ' Save the workbook to the specified filename in Excel 2010 format
            workbook.SaveToFile(result, ExcelVersion.Version2010)

            ' Release the resources used by the workbook
            workbook.Dispose()

            'View the document
            FileViewer(result)
		End Sub

		Private Sub FileViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

		Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnClose.Click
			Close()
		End Sub
	End Class
End Namespace
